src/skins/vector/css/matrix-react-sdk/views/rooms/EventTile.css

```diff
@@ -64,16 +64,12 @@ limitations under the License.
     margin-right: 95px;
 }
 
-/* all the overflow-y: hidden; are to trap Zalgos -
-   but they introduce an implicit overflow-x: auto.
-   so make that explicitly hidden too to avoid random
-   horizontal scrollbars occasionally appearing, like in
+/* Removed hidden overflow in order to have hanging elements
+   in the message stream gutters. This may re-introduce
    https://github.com/vector-im/vector-web/issues/1154
-    */
+*/
 .mx_EventTile_content {
     display: block;
-    overflow-y: hidden;
-    overflow-x: hidden;
```

Comment:
don't we want `overflow: visible;` rather than the default `auto` here to avoid a repeat performance of #1154?